plugins/node/opentelemetry-instrumentation-express/test/express.test.ts

```diff
@@ -455,6 +455,43 @@ describe('ExpressInstrumentation', () => {
         }
       );
     });
+    it('should ignore double slashes in routes', async () => {
+      const rootSpan = tracer.startSpan('rootSpan');
+      let finishListenerCount: number | undefined;
+      let rpcMetadata: RPCMetadata | undefined;
+      const httpServer = await serverWithMiddleware(tracer, rootSpan, app => {
+        app.use(express.json());
+        app.use((req, res, next) => {
+          rpcMetadata = getRPCMetadata(context.active());
+          res.on('finish', () => {
+            finishListenerCount = res.listenerCount('finish');
+          });
```

Comment:
Hmm, is there a specific goal with this and the associated assertion below? :thinking: 
To my untrained eye it seems somewhat unrelated to what's being tested here.

Reply:
Oh good point, looks like extra copy-pasta leftover from another test. Removed